pkg/client/dtclient/settings_client.go

```diff
@@ -257,7 +262,7 @@ func (d *DynatraceClient) upsertSettings(ctx context.Context, obj SettingsObject
 		obj.OriginObjectId = ""
 	}
 
-	payload, err := buildPostRequestPayload(ctx, obj, externalID)
+	payload, err := buildPostRequestPayload(ctx, obj, externalID, options.InsertAfter)
```

Comment:
Do we have any way to warn the user if they've used `insertAfter` for a Setting that is not actually ordered?
If it would mean a dedicated fetching of the schema just for that warning it might not be worth it - but I think when we check for matches by schema constraints we already grab the schema & the client caches the response, right? So it might be possible without notable overhead to read the schema and spit out a log

Reply:
Yes, it's easily possible to ask the cache and spit out a warning, however the API already returns a quite specific error that is logged in this case: ```{"code":400,"message":"insertAfter not supported for unordered schema builtin:rum.web.name"}```

Reply:
It can actually be treated as an error and would save us from retrying the operation a couple of times